src/app/tic-tac-toe.component.ts

```diff
@@ -79,7 +79,7 @@ export class TicTacToeComponent {
       } else if (this.playerTurn === Turn.Second) {
         this.cells[i][j] = Cell.Circle;
         this.playerTurn = Turn.First;
-      }
+      } // review
```

Comment:
Also, don't be scared of the ternary operator! The inner `if-else` could be reduced to 2 assignment statements.